### ISSUE TITLE
Create an installer

### ISSUE THREAD

**Issue by athairus** [team-phoenix/Phoenix#69]
Let's not half-ass this. Make an installer as slick as Phoenix is. Visual Studio has a cool custom installer, why shouldn't we?

**Comment by Druage** [comment]
Of course, having a QML themed installed would be great. I'd just need to learn about the actual installation process.

**Comment by athairus** [comment]
I've acquired a license for BitRock's InstallBuilder, I'll play around with it at some future date.

**Comment by flubbacrud** [comment]
I also have licenses for VMware ThinApp. This is more of a sandboxing tool but does make applications:

1. Portable. Sandbox can be saved out to the launch DIR so a USB drive.
2. It also allows it to be forward compatible without having to change the underlying code to new Windows OS. It also doesn't matter on the system language because it takes care of all of that. Example is on XP %ALLUSERSPROFILE% is now %PROGRAMDATA%. ThinApp uses its own Variables to deal with this so doesn't matter on the version of windows.

I also have InstallShield.
